stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -377,7 +377,7 @@ private int applyOperation() throws IOException {
           mInStream = mFs.open(mFilePath);
         }
         if (mParameters.mReadRandom) {
-          mCurrentOffset = mRandom.nextInt(mMaxOffset);
+          mCurrentOffset = Math.abs(mRandom.nextLong()) % mMaxOffset;
```

Comment:
interesting. Is there no `Random#nextLong(long)` API?

Reply:
Yeah, there isn't. However, I updated this to use the long stream.